server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -5609,9 +5609,7 @@ public VirtualMachine migrateVirtualMachine(Long vmId, Host destinationHost) thr
         }
 
         if (!isOnSupportedHypevisorForMigration(vm)) {
-            if (s_logger.isDebugEnabled()) {
-                s_logger.debug(vm + " is not XenServer/VMware/KVM/Ovm/Hyperv, cannot migrate this VM form hypervisor type " + vm.getHypervisorType());
-            }
+            s_logger.error(vm + " is not XenServer/VMware/KVM/Ovm/Hyperv, cannot migrate this VM form hypervisor type " + vm.getHypervisorType());
```

Comment:
nit picking: the target host can be specified and small spello
```suggestion
            s_logger.error(vm + " is not XenServer/VMware/KVM/Ovm/Hyperv, cannot migrate this VM from hypervisor type " + vm.getHypervisorType() + " to hypervisor type " + destinationHost.getHypervisorType());
```